controllers/subscription.go

```diff
@@ -70,34 +69,28 @@ func newSubscription(p api.Pattern) *operatorv1alpha1.Subscription {
 		},
 	}
 
-	if p.Spec.GitOpsConfig.UseCSV {
-		spec.StartingCSV = p.Spec.GitOpsConfig.OperatorCSV
-	}
-
-	if p.Spec.GitOpsConfig.ManualApproval {
-		spec.InstallPlanApproval = operatorv1alpha1.ApprovalManual
-	}
-
-	return &operatorv1alpha1.Subscription{
+	newSubscription = &operatorv1alpha1.Subscription{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "openshift-gitops-operator",
-			Namespace: subscriptionNamespace,
+			Name:      GitOpsDefaultPackageName,
+			Namespace: SubscriptionNamespace,
 		},
-		Spec: spec,
+		Spec: &newSpec,
 	}
+
+	return newSubscription.DeepCopy(), nil
 }
 
-func getSubscription(client olmclient.Interface, name, namespace string) (error, *operatorv1alpha1.Subscription) {
+func getSubscription(client olmclient.Interface, name, namespace string) (*operatorv1alpha1.Subscription, error) {
 
-	sub, err := client.OperatorsV1alpha1().Subscriptions(subscriptionNamespace).Get(context.Background(), name, metav1.GetOptions{})
+	sub, err := client.OperatorsV1alpha1().Subscriptions(SubscriptionNamespace).Get(context.Background(), name, metav1.GetOptions{})
 	if err != nil {
-		return err, nil
+		return nil, err
 	}
-	return nil, sub
+	return sub, nil
 }
```

Comment:
Same here. A nice consistency cleanup, but either defer it for later, or submit it first

Reply:
Ok. Noted.